docs/languages/csharp/expression-trees.md

```diff
@@ -0,0 +1,132 @@
+﻿# Expression Trees
+Expression Trees are one of the core concepts in computer science language deign.
+In most languages you may have used, expression trees are used by the language
+compiler software as part of the compilation process.
+
+However, many popular languages hide the details of expression trees from the
+developers that use the language.
+
+C# is different.
+
+C# developers can create and interact with Expression Trees.
+
+This article covers the fundamentals of expression trees, how they are created
```

Comment:
Not sure if this is worth calling out at this point (I think we still need to determine our strategy for how Roslyn documentation fits in here), but would this be worth noting how this differs from using Roslyn?  I could see some people getting a little confused over Expression Trees over utilizing Roslyn to get Syntax Trees and Semantic Models for their code.